server/services/store/sqlstore/user.go

```diff
@@ -158,3 +179,44 @@ func (s *SQLStore) UpdateUserPasswordByID(userID, password string) error {
 
 	return nil
 }
+
+func (s *SQLStore) GetUsersByWorkspace(workspaceID string) ([]*model.User, error) {
+	return s.getUsersByCondition(nil)
```

Comment:
Shouldn't this contain the condition. 
```
sq.Eq{"workspaceID": workspaceID}
```

Reply:
This function will only be called for the standalone version of Focalboard, which has just one workspace, qualifying all users as member of that one workspace. Also in Focalboard database, we have no relationship between user and the workspace.

Later on @jespino will be adding a layer of top of this SQL store which would retrieve the Mattermost channel from workspace ID, and then verify user's association with that channel. The `workspaceID` param is only there for the purpose of interface and isn't used in Focalboard's own database.

Reply:
I think we should add a comment here to explain that. It is not clear from the context.